packages/vm/tests/util.js

```diff
@@ -408,3 +409,31 @@ exports.getRequiredForkConfigAlias = function (forkConfig) {
 exports.isRunningInKarma = () => {
   return typeof window !== 'undefined' && window.__karma__
 }
+
+/**
+ * Returns a DAO common which has a different activation block than the default block
+ */
+exports.getDAOCommon = function(activationBlock) {
+  // here: get the default fork list of mainnet and only edit the DAO fork block (thus copy the rest of the "default" hardfork settings)
+  const defaultDAOCommon = new Common('mainnet', "dao")
+  // retrieve the hard forks list from defaultCommon...
+  let forks = defaultDAOCommon.hardforks()
+  let editedForks = []
+  // explicitly edit the "dao" block number:
+  for (let fork of forks) {
+    if (fork.name == "dao") {
+      editedForks.push({
+        name: "dao",
+        forkHash: fork.forkHash,
+        block: activationBlock
+      })
+    } else {
+      editedForks.push(fork)
+    }
+  }
+  const DAOCommon = Common.forCustomChain('mainnet', {
+    hardforks: editedForks
+  }, 'dao') 
+  return DAOCommon
```

Comment:
No blocker either, but just for awareness if this gets necessary on future work: this can be generalized to a `getCommonWithCustomHFBlock()` function by also passing in the HF name. Then this can be used potentially for other use cases or HF transitions as well.

Reply:
This will be addressed in https://github.com/ethereumjs/ethereumjs-vm/pull/849 😄